app/javascript/packages/document-capture/components/acuant-capture.tsx

```diff
@@ -339,6 +339,7 @@ function AcuantCapture(
   const { isMobile } = useContext(DeviceContext);
```

Comment:
I tested this out manually and it mostly worked as expected! However I did see one extra selfie attempt - so if I took 2 pictures, it recorded 3.